src/test/java/com/puppycrawl/tools/checkstyle/MainTest.java

```diff
@@ -91,10 +91,12 @@
         + " -J,--treeWithJavadoc                    Print full Abstract Syntax Tree of the file%n"
         + " -o <arg>                                Sets the output file. Defaults to stdout%n"
         + " -p <arg>                                Loads the properties file%n"
+        + " -s <arg>                                Generates basic suppression xpath%n"
```

Comment:
@romani It is not clear from this output what the argument is suppose to be.
Shouldn't we extend the description to specify the format(s) we are expecting?

Reply:
yes, as minimum as "Print sample suppression xpath for all violations"
@rnveach , please suggest better wording.

Reply:
First sentence suggestion is at https://github.com/checkstyle/checkstyle/pull/5464#discussion_r165853342 .
This line was more about adding a description to describe what `arg` is.
There are no examples to go by as other arguments are self-explanator.

My idea right now is: `arg is the line and column number (separated by a :) in the file that the suppression should be generated for`.

Reply:
> arg is the line and column number (separated by a : ) in the file that the suppression should be generated for

Good example. Easy to understand as for me.